sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Infrastructure/DisposableDocumentClassifier.cs

```diff
@@ -22,17 +22,16 @@ internal class DisposableDocumentClassifier : IAsyncDisposable
         /// </summary>
         /// <param name="adminClient">The client to use for deleting the classifier upon disposal.</param>
         /// <param name="classifierId">The identifier of the classifier to delete upon disposal.</param>
-        private DisposableDocumentClassifier(DocumentModelAdministrationClient adminClient, string classifierId)
+        private DisposableDocumentClassifier(DocumentModelAdministrationClient adminClient, DocumentClassifierDetails value)
         {
             _adminClient = adminClient;
-            ClassifierId = classifierId;
+            Value = value;
         }
 
         /// <summary>
-        /// The identifier of the classifier this instance is associated with. It will be deleted upon
-        /// disposal.
+        /// The classifier this instance is associated with. It will be deleted upon disposal.
         /// </summary>
-        public string ClassifierId { get; }
+        public DocumentClassifierDetails Value { get; }
```

Comment:
Updating this class helper to include the whole result returned by the service instead of only storing the classifier ID.